Library/Homebrew/cmd/update.sh

```diff
@@ -372,15 +372,20 @@ EOS
   if [[ -n "$HOMEBREW_FORCE_BREWED_CURL" &&
       ! -x "$HOMEBREW_PREFIX/opt/curl/bin/curl" ]]
   then
-    brew install curl
+    ohai "Installing Homebrew cURL"
+    brew install curl || odie "Failed to install cURL"
```

Comment:
Could also just `set -e` (if it's not already).

Reply:
I added `set -e` and `set +e` before and after the region that installs cURL and git. I'm a bit afraid to set it "globally" in this function.